templates/landing/app_layout.html

```diff
@@ -0,0 +1,41 @@
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
In case it s a layout for all the html files
it should be in higher hierarchy for other tamplates 

── templates
    └── app_layout.html
   ├── landing
   │   └──  homepage.html
   └── registration
       └── login.html